crates/header-translator/translation-config.toml

```diff
@@ -827,3 +830,24 @@ skipped = true
 skipped = true
 [fn.MTLPackedFloat3Make]
 skipped = true
+
+[class.WebFrame.methods]
+# Needs `JSGlobalContextRef` from JavaScriptCore
+globalContext = { skipped = true }
+
+# error: translator assertion failure: invalid error nullability (left: Unspecified) (Right: Nullable)
+[class.WebHistory.methods]
+loadFromURL_error = { skipped = true }
+saveToURL_error = { skipped = true }
+
+[protocol.WebPlugInViewFactory.methods]
+# error: class methods are not supported in `extern_protocol!`
+plugInViewWithArguments = { skipped = true }
```

Comment:
Nit: Move this next to the ones in https://github.com/madsmtm/objc2/blob/2bd99a848d6ea31d279b8dd88e3e1e6a1467b58a/crates/header-translator/translation-config.toml#L321-L339